fvm/Cargo.toml

```diff
@@ -50,6 +50,7 @@ fvm = { path = ".", features = ["testing"], default-features = false }
 version = "1.0.1"
 default-features = false
 features = ["cranelift", "pooling-allocator", "parallel-compilation"]
+quickcheck = "1"
```

Comment:
This might be a copy&paste mistake. In any case, you haven't touched the `fvm` crate so this line shouldn't be here.

Reply:
The ActorState changes were going to be apart of this PR, opted to make it separate after this PR gets through. So yes, it is a copy paste mistake. Removing